docs/how-to/drbd-loader.md

```diff
@@ -0,0 +1,128 @@
+# How to Configure the DRBD Module Loader
+
+This guide shows you how to configure various aspects of the DRBD® Module Loader.
+
+To follow the steps in this guide, you should be familiar with editing `LinstorSatelliteConfiguration` resources.
+
+DRBD Module Loader is the component responsible for making the DRBD kernel module available, as well as loading other
+useful kernel modules for Piraeus.
+
+The following modules are loaded if available:
+
+| Module                    | Purpose                                          |
+|---------------------------|--------------------------------------------------|
+| `libcrc32c`               | dependency for DRBD                              |
+| `nvmet_rdma`, `nvme_rdma` | LINSTOR NVME layer                               |
+| `loop`                    | LINSTOR when using loop devices as backing disks |
+| `dm_writecache`           | LINSTOR writecache layer                         |
+ | `dm_cache`                | LINSTOR cache layer                              |
+ | `dm_thin_pool`            | LINSTOR thin-provisioned storage                 |
+ | `dm_snapshot`             | LINSTOR Snapshots                                |
+| `dm_crypt`                | LINSTOR encrypted volumes                        |
+
+## Disable the DRBD Module Loader
+
+In some circumstances it might be necessary to disable the DRBD Module Loader entirely. For example, you are using an
+immutable operating system, and DRBD and other modules are loaded as part of the host configuration.
+
+To disable the DRBD Module Loader completely, use the following configuration:
+
+```yaml
+apiVersion: piraeus.io/v1
+kind: LinstorSatelliteConfiguration
+metadata:
+  name: no-loader
+spec:
+  patches:
+    - target:
+        kind: Pod
+        name: satellite
+      patch: |
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: satellite
+        spec:
+          initContainers:
+          - name: drbd-module-loader
+            $patch: delete
+```
+
+## Select a Different DRBD Loader Version
+
+By default, the Operator will try to find a DRBD Module Loader matching the host operator system. The host distribution
+is determined by inspecting the `.status.nodeInfo.osImage` field of the Kubernetes Node resource.
```

Comment:
I think it would be helpful to add a sentence along the lines of "A user-defined image can be used if the automatic mapping does not succeed or if you have different module loading requirements." That explains why this information about the default image is followed by the steps below.